verto/tests/InteractiveContainerTest.py

```diff
@@ -402,6 +410,66 @@ def test_custom_arguments_parameters_and_thumbnail_true(self):
         expected_string = self.read_test_file(self.processor_name, 'parameters_and_thumbnail_true_expected.html', strip=True)
         self.assertEqual(expected_string, converted_test_string)
 
+    def test_default_thumbnail_in_required_files(self):
+        '''Test the thumbnail for a whole page interactive is required.'''
+        verto_extension_default = VertoExtension(
+            processors=[self.processor_name],
+        )
+        test_string = self.read_test_file(
+            self.processor_name, 'whole_page_without_thumbnail_parameter.md')
+        converted_test_string = markdown.markdown(
+            test_string, extensions=[verto_extension_default])
```

Comment:
inconsistent syntax for providing multiple paramters. In verto the closing parenthesis is usually given on a new line where multiple lines have to be used for multiple and/or long parameter names, as below:
```
converted_test_string = markdown.markdown(
            test_string, extensions=[verto_extension_default]
)
```
I see this mistake in a few different places where you've added tests, just highlighting it here.

Reply:
Urgh! This is my editor trying to be clever when I copy and paste. I really need to disable that feature!

Thanks for spotting it, I'll fix them today.